translations/ja.json

```
@@ -298,58 +298,58 @@
-	"If true, only download and cache files for later use - don't install or start anything.": "true の場合、後で使用できるようにファイルのダウンロードとキャッシュ保存だけが行われます。インストールも起動も行われません",
-	"If true, pods might get deleted and restarted on addon enable": "",
-	"If true, returns list of profiles faster by skipping validating the status of the cluster.": "",
-	"If true, the added node will be marked for work. Defaults to true.": "",
-	"If true, the node added will also be a control plane in addition to a worker.": "",
-	"If true, will perform potentially dangerous operations. Use with discretion.": "",
-	"If you are running minikube within a VM, consider using --driver=none:": "",
-	"If you are still interested to make {{.driver_name}} driver work. The following suggestions might help you get passed this issue:": "",
-	"If you don't want your credentials mounted into a specific pod, add a label with the `gcp-auth-skip-secret` key to your pod configuration.": "",
-	"If you want existing pods to be mounted with credentials, either recreate them or rerun addons enable with --refresh.": "",
-	"Ignoring empty custom image {{.name}}": "",
-	"Ignoring invalid pair entry {{.pair}}": "",
-	"Ignoring unknown custom image {{.name}}": "",
-	"Ignoring unknown custom registry {{.name}}": "",
+	"Hide the hypervisor signature from the guest in minikube (kvm2 driver only)": "minikube 中のゲストに対してハイパーバイザー署名を非表示にします (kvm2 ドライバーのみ)",
+	"Hyperkit is broken. Upgrade to the latest hyperkit version and/or Docker for Desktop. Alternatively, you may choose an alternate --driver": "Hyperkit は故障しています。最新バージョンの Hyperkit と Docker for Desktop にアップグレードしてください。あるいは、別の --driver を選択することもできます。",
+	"Hyperkit networking is broken. Try disabling Internet Sharing: System Preference \u003e Sharing \u003e Internet Sharing. \nAlternatively, you can try upgrading to the latest hyperkit version, or using an alternate driver.": "Hyperkit ネットワーキングは故障しています。インターネット共有の無効化を試してください: System Preference \u003e Sharing \u003e Internet Sharing 。\nあるいは、最新の Hyperkit バージョンへのアップグレードか、別のドライバー使用を試すこともできます。",
+	"IP Address to use to expose ports (docker and podman driver only)": "ポートの expose に使用する IP アドレス (docker, podman ドライバーのみ)",
+	"IP address (ssh driver only)": "IP アドレス (SSH ドライバーのみ)",
+	"If present, writes to the provided file instead of stdout.": "指定すると、標準出力の代わりに指定されたファイルに出力します。",
+	"If set, automatically updates drivers to the latest version. Defaults to true.": "設定すると、自動的にドライバーを最新バージョンに更新します。デフォルトは true です。",
+	"If set, delete the current cluster if start fails and try again. Defaults to false.": "設定すると、現在のクラスターの起動に失敗した場合はクラスターを削除して再度試行します。デフォルトは false です。",
+	"If set, disables optimizations that are set for local Kubernetes. Including decreasing CoreDNS replicas from 2 to 1 and increasing kubeadm housekeeping-interval from 10s to 5m. Defaults to false.": "設定すると、最適化された設定をローカルの Kubernetes 用に無効化します。CoreDNS レプリカ数を 1 に減らし、kubeadm のハウスキーピング間隔を 10 秒から 5 分に増加させることを含みます。デフォルトは false です。",
```

```suggestion
	"If set, disables optimizations that are set for local Kubernetes. Including decreasing CoreDNS replicas from 2 to 1 and increasing kubeadm housekeeping-interval from 10s to 5m. Defaults to false.": "設定すると、ローカルの Kubernetes 用に設定された最適化を無効化します。CoreDNS レプリカ数を 2 から 1 に減らし、kubeadm の housekeeping-interval を 10 秒から 5 分に増加させることを含みます。デフォルトは false です。",
```

- "ローカルの Kubernetes 用に最適化された設定を無効化" is better translation I think.
- Translation of "from 2" is missing.
- `housekeeping-interval` is an option parameter for kubeadm, so it's better to keep original text.